README.md

```diff
@@ -48,3 +48,13 @@ Use it in your anywhere in your application
 ````
 <ng-month-picker [(ngModel)]="currentMonth"></ng-month-picker>
 ````
+
+Use it within a form:
```

Comment:
Could you please change this to **Example for template-driven form**? You can also add an example of using it with reactive forms.

Reply:
Done.